benchmark/single-source/ReversedCollections.swift

```diff
@@ -13,58 +13,61 @@
 import TestsUtils
 
 public let ReversedCollections = [
-  BenchmarkInfo(name: "ReversedArray", runFunction: run_ReversedArray, tags: [.validation, .api, .Array]),
+  BenchmarkInfo(name: "ReversedArray2", runFunction: run_ReversedArray, tags: [.validation, .api, .Array],
```

Comment:
I don't think renaming ReversedCollections is necessary. You are not changing the workload, just fixing a bug in test. (Unless it was meant as a technique to highlight them for review in the benchmark report...)